packages/taxdat/R/file_name_functions.R

```diff
@@ -39,7 +63,7 @@ make_covar_filename <- function(cholera_directory,
 #' @param map_name map name
 #' @param covariate_name_part name of covariate
 #' @param config configuration file
```

Comment:
can we clarify here that these are the actual config and config_dict objects and not the file names?objects an